src/views/ParticipantEntity/FamilyTable/index.tsx

```diff
@@ -41,6 +45,24 @@ const FamilyTable = ({ participant, loading }: OwnProps) => {
           dispatch(
             updateUserConfig({ participants: { tables: { family: { columns: newState } } } }),
           ),
+        onTableExportClick: () =>
+          dispatch(
+            fetchTsvReport({
+              columnStates: userInfo?.config.participants?.tables?.family?.columns,
```

Comment:
ou tu es sur si userInfo existe il y a forcement une config dedans ?
```suggestion
              columnStates: userInfo?.config?.participants?.tables?.family?.columns,
```